vitrina/datasets/templates/vitrina/datasets/members_list.html

```diff
@@ -0,0 +1,279 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block pageTitle %} | {% translate "Narių sąrašas" %}{% endblock %}
+{% block pageOgTitle %} | {% translate "Narių sąrašas" %}{% endblock %}
+
+{% block head %}
+    <style>
+        .autocomplete {
+            /*the container must be positioned relative:*/
+            position: relative;
+            display: inline-block;
+        }
+
+        .autocomplete-items {
+            position: absolute;
+            border: 1px solid #d4d4d4;
+            border-bottom: none;
+            border-top: none;
+            z-index: 99;
+            /*position the autocomplete items to be the same width as the container:*/
+            top: 100%;
+            left: 0;
+            right: 0;
+            color: #4a4a4a;
+        }
+
+        .autocomplete-items div {
+            padding: 10px;
+            cursor: pointer;
+            background-color: #fff;
+            border-bottom: 1px solid #d4d4d4;
+        }
+
+        .autocomplete-items div:hover {
+            /*when hovering an item:*/
+            background-color: #e9e9e9;
+        }
+
+        .autocomplete-active {
+            /*when navigating through the items using the arrow keys:*/
+            background-color: #007db2 !important;
+            color: #ffffff;
+        }
+    </style>
+{% endblock %}
+
+{% block current_title %}
+{% translate org.title %}
+{% translate "organizacijos narių sąrašas" %}
+{% endblock %}
+
+{% block content %}
+    <table class="table is-fullwidth has-border-top has-border-bottom th-40 custom-table-padding">
+        <tbody>
+        {% for i in page_obj %}
+            <tr>
+                <table class="table is-fullwidth has-border-top has-border-bottom th-40 custom-table-padding">
+                    <th>{% translate "Narys" %}</th>
+                    <td>{{ i.user.first_name }} {{ i.user.last_name }}</td>
+                    <tbody>
+                        <tr>
+                            <th>{% translate "Nario rolė"%}</th>
+                            <td>{{ i.role }}</td>
+                        </tr>
+                        <tr>
+                            <th>{% translate "Organizacija"%}</th>
+                            <td>{{ i.organization.title }}</td>
+                        </tr>
+                        <tr style="margin-bottom:25px;padding-bottom:25px">
+                            <th>{% translate "Nario sukūrimo data"%}</th>
+                            <td>{{ i.created }}</td>
+                        </tr>
+                    </tbody>
+                </table>
+            </tr>
+        {% endfor %}
+        </tbody>
+    </table>
```

Comment:
The view does support pagination, but this template doesn't. If you handle pagination in the view, please add it to the template too.